src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -188,30 +188,44 @@ public override ExportResult Export(in Batch<Metric> batch)
                 }
 
                 var exemplarString = new StringBuilder();
-                foreach (var exemplar in metricPoint.GetExemplars())
+                if (metricPoint.TryGetExemplars(out var exemplars))
                 {
-                    if (exemplar.Timestamp != default)
+                    foreach (ref readonly var exemplar in exemplars)
                     {
-                        exemplarString.Append("Value: ");
-                        exemplarString.Append(exemplar.DoubleValue);
-                        exemplarString.Append(" Timestamp: ");
+                        exemplarString.Append("Timestamp: ");
```

Comment:
nit: Timestamp is optional, so may need to omit this if default

Reply:
Wait how is timestamp optional? If we have a `default` timestamp we now don't even return the `Exemplar` from `ReadOnlyExemplarCollection` so if this is the case we'll need to change that 🔨

Reply:
sorry my bad. I incorrectly assumed tiemstamp was option! false alarm.